app/host.json

```diff
@@ -0,0 +1,20 @@
+{
+  "version": "2.0",
+  "logging": {
+    "applicationInsights": {
+      "samplingSettings": {
+        "isEnabled": true,
+        "excludedTypes": "Request"
+      }
+    }
+  },
+  "extensions": {
+    "durableTask": {
+      "hubName": "%TASKHUB_NAME%",
```

Comment:
Unless there's a specific reason to make this an app setting, I recommend just hardcoding the task hub name in the host.json file. Hardcoding it reduces the likelihood of making a mistake. I don't think using an app setting should necessarily be considered a best practice.

Reply:
Yeah, I recommend changing this in samples to make them easier to run.